src/TwigYard/Component/Application.php

```diff
@@ -169,25 +180,29 @@ private function getDefaultSiteParameters()
     }
 
     /**
+     * @param ContainerBuilder $containerBuilder
      * @param array $parameters
-     * @return \TwigYard\Component\ConfigCache
+     * @return ConfigCache
      */
-    private function getConfigCache(array $parameters)
+    private function registerConfigCacheService(ContainerBuilder $containerBuilder, array $parameters)
```

Comment:
Lets try to use factory methods for this.
https://symfony.com/doc/current/service_container/factories.html

Reply:
Fixed. I have also changed LoggerFactory to be able to be registered in services.yml file.